api/http/saml.go

```diff
@@ -60,9 +60,9 @@ type SamlSLORequestHandler struct {
 
 // ServeHTTP is the initial entry point for authentication.
 func (service SamlSLORequestHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
-	if !service.Env.True(api.SamlEnabled) {
+	if !service.Env.True(api.SamlSloEnabled) {
 		service.Log.Warn(api.SamlAttemptDenied, api.LogFields{})
-		http.Error(w, "SAML is not implemented", http.StatusInternalServerError)
+		http.Error(w, "SAML SLO is not implemented", http.StatusInternalServerError)
```

Comment:
In this case, it looks like we log a generic error message but send a more detailed reason back to caller. Should be opposite (or the same detailed message). Also should say "enabled" vs "implemented".